models.py

```diff
@@ -13,6 +13,8 @@
 class EVC(GenericEntity):
     """Class that represents a E-Line Virtual Connection."""
 
+    unique_attributes = ['name', 'uni_a', 'uni_z']
```

Comment:
Unique means they can't be duplicated, but the main thing here is to protect them from being updated, right? Then I'd suggest changing the name to private or protected attributes.